packages/git/src/browser/history/git-history-contribution.ts

```diff
@@ -30,7 +31,9 @@ export const GIT_HISTORY_MAX_COUNT = 100;
 export class GitHistoryContribution extends AbstractViewContribution<GitHistoryWidget> {
 
     constructor(
-        @inject(SelectionService) protected readonly selectionService: SelectionService) {
+        @inject(SelectionService) protected readonly selectionService: SelectionService,
+        @inject(GitRepositoryTracker) protected readonly repositoryTracker: GitRepositoryTracker
```

Comment:
Maybe just make these constructor parameters be attributes, in order to comply with https://github.com/theia-ide/theia/wiki/Coding-Guidelines#dependency-injection ? @akosyakov is the idea to refactor when we can ?

Reply:
you are right. i should have complied with the documented standards.